src/commons/components/catalog-content-item/catalog-content-item.tsx

```diff
@@ -3,7 +3,19 @@ import React from 'react';
 // CSS
 import './catalog-content-item.css';
 
-function CatalogContentItem(props) {
+interface CatalogContentItemProps {
+  id: number,
+  melody: CatalogContentItemPropsMelody
+}
+
+interface CatalogContentItemPropsMelody {
+  img: any,
+  musicName: string,
+  artist: string,
+  price: number,
+}
+
+function CatalogContentItem(props: CatalogContentItemProps) {
```

Comment:
fix it 

Example:
const CatalogGenres: React.FunctionComponent<CatalogGenresProps> = ({genres, isActiveGenre, genreClick}) => {